packages/vuetify/src/components/VTimePicker/VTimePicker.ts

```diff
@@ -197,6 +201,17 @@ export default mixins(
       const value = this.genValue()
       if (value !== null) this.$emit('input', value)
     },
+    emitPicker (value: SelectingTimes) {
+      let val = 'HOUR'
+      if (value === SelectingTimes.Minute) val = 'MINUTE'
+      else if (value === SelectingTimes.Second) val = 'SECOND'
+      this.$emit('update:active-picker', val)
+    },
+    setPicker (picker: ActivePicker) {
+      if (picker === 'HOUR') this.selecting = SelectingTimes.Hour
```

Comment:
please add brackets (also in `emitPicker`)
```js
if (...) {
  ...
} else if (...) {
  ...
}
```

Reply:
done